lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
> That's your proposal, I believe.

If I'm reading the code right, [it's already so](https://github.com/wikimedia/restbase/blob/56a45db06ab04820b1e3612183faf6109422e2a1/lib/reqTemplate.js#L264)